yt_dlp/extractor/yourreformer.py

```diff
@@ -0,0 +1,85 @@
+from .common import InfoExtractor
+
+
+class YourReformerIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?yourreformerathome\.uscreen\.io/programs/(?P<id>[^/?#&]+)'
+    _TESTS = [{
+        'url': 'https://yourreformerathome.uscreen.io/programs/arms-back-abs-7-with-erin?category_id=139586',
+        'info_dict': {
+            'id': 'arms-back-abs-7-with-erin',
+            'ext': 'mp4',
+            'title': 'Arms, Back & Abs 7',
+        },
+        'params': {
+            'skip_download': True,
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        # Extract category_id from the URL
+        category_id = None
+        if '?' in url:
+            query = url.split('?', 1)[1]
+            for param in query.split('&'):
+                if param.startswith('category_id='):
+                    category_id = param.split('=', 1)[1]
+                    break
+
+        # Construct the program_content URL
+        program_content_url = f'https://yourreformerathome.uscreen.io/programs/{video_id}/program_content'
+        query_params = {
+            'category_id': category_id,
+            'playlist_position': 'sidebar',
+            'preview': 'false',
+        }
+
+        # Request headers for both requests
+        headers = {
+            'accept': '*/*',
+            'accept-language': 'en-AU,en-GB;q=0.9,en-US;q=0.8,en;q=0.7',
+            'cache-control': 'no-cache',
+            'origin': 'https://yourreformerathome.uscreen.io',
+            'pragma': 'no-cache',
+            'priority': 'u=1, i',
+            'referer': 'https://yourreformerathome.uscreen.io/',
+            'sec-ch-ua': '"Not(A:Brand";v="99", "Google Chrome";v="133", "Chromium";v="133"',
+            'sec-ch-ua-mobile': '?0',
+            'sec-ch-ua-platform': '"macOS"',
+            'sec-fetch-dest': 'empty',
+            'sec-fetch-mode': 'cors',
+            'sec-fetch-site': 'cross-site',
+            'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/133.0.0.0 Safari/537.36',
```

Comment:
remove unnecessary headers. You probably only need the referer (if at all)

this also only appears to be used for the stream download and not the initial webpage download. can you confirm what is required.